### DISCUSSION TITLE
Search returns duplicated (3x) results

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#997] [General]
When I do a search, I get duplicated results which point to the same tour but with three different Lucerne doc ID's.  This is true for any and all searches regardless of search target.
Is there a way to rebuild the Lucerne doc database indices (assuming that is the problem)?

Below image capture shows search returning 2 different tours but 2*3=6 results
![MTB Search Duplicates](https://user-images.githubusercontent.com/106929478/219715347-7a31e7f2-8015-472e-8950-b5054eb09e5e.jpg)

**Comment by wolfgang-ch** [comment]
Fulltext issues can be fixed by deleting it's index, see here https://github.com/mytourbook/mytourbook/issues/860#issuecomment-1124527192

**Comment by wolfgang-ch** [comment]
In the next version there will be a hint in the documentation https://mytourbook.sourceforge.io/mytourbook/index.php/documentation/show-tours/search#fulltext-issues

![image](https://user-images.githubusercontent.com/1283445/219844244-0040ff34-b8ea-476b-8f25-bc98dc2eb5d3.png)

**Comment by stvzldn** [comment]
Perfect - delete of the lucerne-index folder resolved the issue, thank you.
